src/app/features/courses/courses-list/courses-list.component.html

```diff
@@ -0,0 +1,9 @@
+<main class="course-list">
+  <app-course-card
+    *ngFor="let course of courses"
```

Comment:
Techinally this is correct, however I would encourage you to move the iteration up a level. An ng-container might do just fine, also the variable `course` defined there will be available for child elements. This will increase readability for `app-course-card` as it receives a single Course item as an input

Reply:
done!